src/util/config/config.cpp

```diff
@@ -921,6 +921,13 @@ namespace dxvk {
     { R"(\\(nations|patriots)\.exe$)", {{
       { "d3d8.managedBufferPlacement",     "False" },
     }} },
+    /* V-Rally 3                                  */
+    { R"(\\VRally3Demo\.exe$)", {{
```

Comment:
Probably a good idea to just do R"(\\VRally3(Demo)?\.exe$)" and match both the game and demo in a single entry (there are several examples of similar situations above, like Evil Within). Welcome to the wondrous world of regex :).

Reply:
Agreed, that works out better.